kingpin/constants.py

```diff
@@ -46,3 +48,18 @@ class STATE(StringCompareBase):
     """
 
     valid = ('present', 'absent')
+
+
+class SchemaCompareBase(object):
```

Comment:
What do you think about removing the check for `valid` parameter and always relying on `validate` method? The STATE would simply have `validate(self, option): return option in ('present', 'state')`